### PR TITLE
Create initial version of the specification

### PR DESCRIPTION
**Pull request by mounirlamouri**
Still a couple of TODO to resolve but it's close enough. PTAL

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+    </pre>
+
+    <p>
+      The {{MediaKeysPolicy}} dictionary is an object consisting of only
+      optional properties. Each property represents a <dfn>policy requirement</dfn>.
+    </p>
+
+    <p>
+      A <a>policy requirement</a> is said to be <dfn>fulfilled</dfn> if the
+      system match the requirements. The exact requirements to match each
```
Comment (joeyparrish) [inline index.bs]: grammar nit: "if the system _matches_ the requirements"

File: index.bs [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+      <a>policy requirement</a> is defined below.
```
Comment (joeyparrish) [inline index.bs]: grammar nit: "_are_ defined below" (so the verb agrees with the subject "requirements")

File: index.bs [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+    partial interface MediaKeys {
+      [NewObject] Promise&lt;MediaKeyStatus&gt; getStatusForPolicy(optional MediaKeysPolicy policy);
+    };
+  </pre>
+
+  <p>
+    The {{getStatusForPolicy}} method MUST run the following steps:
+      <ol>
+        <li>
+          If <var>policy</var> has no <a>present</a> <a>member</a>, return a
```
Comment (joeyparrish) [inline index.bs:122]: What is "present" here?  Is that supposed to be the name of a member?  Or is this meant to say that "policy" can't be empty?

File: index.bs [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+      <p>
+        The API offers access to some information that can already be accessed
+        by the page through the CDM. This information does not increase the
+        fingerprint surface but can make it simpler for a website to fingerprint
+        its users as the information is easier to access and no longer requires
+        getting a key from the CDM.
+      </p>
+
+      <p>
+        The information added by this API, regardless of it already be
```
Comment (joeyparrish) [inline index.bs]: grammar nit: ", regardless of whether or not it is accessible in other ways, "

File: index.bs [hunk 5]
```diff
@@ -0,0 +1,209 @@
...
+        accessible via other ways does not increase the entropy for
+        fingerprinting much as HDCP is widely available on most operating
+        systems and displays. The difference will lie on the edges that will
```
Comment (joeyparrish) [inline index.bs]: I'm not sure what is meant by "lie on the edges".  Maybe we can rephrase this somehow.

File: index.bs [hunk 6]
```diff
@@ -0,0 +1,209 @@
...
+    <section>
+      <h3 id='user-consent'>
+        User Consent
+      </h3>
+
+      <p>
+        The fingerprinting concerns are mitigated by the fact that the API is
+        only accessible after <code>requestMediaKeySystemAccess()</code> was
+        called which may require user consent. UAs that would require user
+        consent in order to access the CDM will therofer require user consent
```
Comment (joeyparrish) [inline index.bs]: typo: "therofer" => "therefore"

**Review by joeyparrish — CHANGES_REQUESTED** [review]
Looks great!  I see a few small grammar issues, but nothing major.  Thank you for writing this!

File: index.bs (hunk 3 above)
Comment (mounirlamouri) [reply index.bs:122]: In a dictionary, a member is present if it was set. So in `{ foo: '' }` `foo` is present but `bar` is not. IOW, the idea is that if you pas `{}`, it doesn't check for `minHdcpPolicy` being fulfilled.

File: index.bs (hunk 5 above)
Comment (mounirlamouri) [reply index.bs]: Drop the entire sentence. Didn't add much.

**Comment by mounirlamouri** [comment]
PTAL

**Review by joeyparrish — APPROVED** [review]
Thanks!

**Pull request merged**